### PR TITLE
fix: update logic when an employee get physical resources

### LINKED ISSUE #276: fix: employee constraint
some endpoints (get all import requests, get all borrow requests,...) has a constraint where employee has to include room Id to access that resource, which does not make sense. please remove it and instead check for if roomId exists then that room should be in the user's department

### PR DESCRIPTION
**Pull request by kaitoz11**

### CODE REVIEW

File: src/Application/Borrows/Queries/GetAllBorrowRequestsPaginated.cs [hunk 1]
```diff
@@ -93,6 +70,24 @@ public QueryHandler(IApplicationDbContext context, IMapper mapper)
                 .ThenInclude(t => t.Room)
                 .ThenInclude(s => s.Department);
 
+            if (request.CurrentUser.Role.IsEmployee())
+            {
+                borrows = borrows.Where(x => x.CreatedBy! == request.CurrentUser.Id);
```
Comment (ChienNQuang) [inline src/Application/Borrows/Queries/GetAllBorrowRequestsPaginated.cs]: where is the check EmployeeId == CurrentUser.Id? You removed it?

**Review by ChienNQuang — CHANGES_REQUESTED** [review]
Over-removal

File: src/Application/Borrows/Queries/GetAllBorrowRequestsPaginated.cs (hunk 1 above)
Comment (kaitoz11) [reply src/Application/Borrows/Queries/GetAllBorrowRequestsPaginated.cs]: fixed at https://github.com/ProFile-org/ProFile-BE/pull/309/commits/c41717494ebcf25bc46e19be089faa4a1539a994

**Review by ChienNQuang — APPROVED** [review]
hehe

**Pull request merged**